### ISSUE TITLE
(core dumped) ./scripts/elementaryplus-installer

### ISSUE THREAD

**Issue by jbreitbart** [mank319/elementaryPlus#42]
I just cloned the current repository, but install.sh crashes with:

```
$ ./install.sh 

(elementaryplus-installer:28152): GLib-GIO-ERROR **: Settings schema 'apps.elementaryPlusConfigurator' is not installed

./install.sh: line 1: 28152 Trace/breakpoint trap   (core dumped) ./scripts/elementaryplus-installer
```

**Comment by jbreitbart** [comment]
I fixed it by manually calling

$ sudo scripts/first_start.sh /home/jbreitbart/temp/elementaryPlus
First run: done!

and remove the pkexec call from the python script. Not sure why this is happening.

**Comment by cybre** [comment]
Sorry for the late response. I can't reproduce the bug on my system, didn't you get an authentication dialog when you first started the script? If not, did you try installing some of the indicator icons? Did the installer prompt you for you password then?

**Comment by jbreitbart** [comment]
No, there was no authentification dialog, just the core dump. There was no GUI showing, so I could not do anything and did not try to install some indicator icons.

**Comment by bil-elmoussaoui** [comment]
@jbreitbart can you provide more informations about this? i would like to try to fix this

**Comment by cybre** [comment]
Fixed #114
